examples/lunr/lunr.js

```diff
@@ -35,7 +35,7 @@ var hitTemplate = (hit, i) => `
   </div>
   <button objectID="${hit.objectID}" position="${i +
```

Comment:
You can change their name to be consistent with the others: `data-xxx`.

Reply:
Thanks, missed that one!
i've added a commit to tackle it
also fixed `position -> data-position`